src/ensembl/production/datacheck/static/img/production.svg

```diff
@@ -0,0 +1,534 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This image is the Production logo that I designed for a retreat a few years ago (there always used to be some sort of creative task set pre-retreat, which ended up on the mug, or whatever swag, that you got to commemorate the occasion).
However, it's a bit small to use as a favicon; I recently reworked the logo into something a bit bolder for the metadata refactoring: https://github.com/Ensembl/ensembl-prodinf-metadata/tree/main/src/ensembl/production/metadata/static/img
We don't have to use that in the long term, by any means, but it probably works better than this image...